radarr/radarr.go

```diff
@@ -431,3 +431,22 @@ func (r *Radarr) SendCommand(cmd *CommandRequest) (*CommandResponse, error) {
 
 	return &output, nil
 }
+
+// Lookup will search for movies matching the specified search term.
+func (r *Radarr) Lookup(term string) ([]Movie, error) {
```

Comment:
Does this work OK with the `[]Movie` type? Sometimes there are variations in new methods that must be accounted for. I use this tool a lot to check the consistency of my structs vs the json. https://mholt.github.io/json-to-go/

Reply:
Here is the generated struct from the response body of this endpoint. It matches `Movie`. 
```go
type AutoGenerated []struct {
	ID              int    `json:"id"`
	Title           string `json:"title"`
	SortTitle       string `json:"sortTitle"`
	SizeOnDisk      int    `json:"sizeOnDisk"`
	Overview        string `json:"overview"`
	InCinemas       string `json:"inCinemas"`
	PhysicalRelease string `json:"physicalRelease"`
	Images          []struct {
		CoverType string `json:"coverType"`
		URL       string `json:"url"`
		RemoteURL string `json:"remoteUrl"`
	} `json:"images"`
	Website             string   `json:"website"`
	Year                int      `json:"year"`
	HasFile             bool     `json:"hasFile"`
	YouTubeTrailerID    string   `json:"youTubeTrailerId"`
	Studio              string   `json:"studio"`
	Path                string   `json:"path"`
	RootFolderPath      string   `json:"rootFolderPath"`
	QualityProfileID    int      `json:"qualityProfileId"`
	Monitored           bool     `json:"monitored"`
	MinimumAvailability string   `json:"minimumAvailability"`
	IsAvailable         bool     `json:"isAvailable"`
	FolderName          string   `json:"folderName"`
	Runtime             int      `json:"runtime"`
	CleanTitle          string   `json:"cleanTitle"`
	ImdbID              string   `json:"imdbId"`
	TmdbID              int      `json:"tmdbId"`
	TitleSlug           string   `json:"titleSlug"`
	Certification       string   `json:"certification"`
	Genres              []string `json:"genres"`
	Tags                []int    `json:"tags"`
	Added               string   `json:"added"`
	Ratings             struct {
		Votes int `json:"votes"`
		Value int `json:"value"`
	} `json:"ratings"`
	Collection struct {
		Name   string `json:"name"`
		TmdbID int    `json:"tmdbId"`
		Images []struct {
			CoverType string `json:"coverType"`
			URL       string `json:"url"`
			RemoteURL string `json:"remoteUrl"`
		} `json:"images"`
	} `json:"collection"`
	Status string `json:"status"`
}
```

Reply:
awesome. love it when that works out!